src/Gizmos/babylon.axisScaleGizmo.ts

```diff
@@ -5,6 +5,15 @@ module BABYLON {
     export class AxisScaleGizmo extends Gizmo {
         private _dragBehavior:PointerDragBehavior;
         private _pointerObserver:Nullable<Observer<PointerInfo>> = null;
+        /**

```

Comment:
could the code be factorized with the previous one ? it sounds similar with the same dispose comment